collector-bitmovin-player/build.gradle

```diff
@@ -3,32 +3,27 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.jfrog.artifactory'
 apply plugin: 'maven-publish'
-
-def packageName = 'com.bitmovin.analytics'
-def libraryVersion = rootProject.ext.version
+apply from: '../bitmovinpropertiesloader.gradle'
 
 android {
-
-    compileSdkVersion 27
+    compileSdkVersion project.compileSdkVersion
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 27
-        versionCode rootProject.ext.versionCode
-        versionName libraryVersion
+        minSdkVersion project.minSdkVersion
```

Comment:
Are we sure we want to have this centralized? The library might have a different mindSdk version than the app so we can test version mismatches and stuff (e.g the customer is on 20 and our collector is built with `16` etc)

Reply:
we should have it centralised, as we want to have 3x modules (`collector`, `collector-bitmovin-player`, `collector-exoplayer`) written in way to support this minSDK 16.

If it is needed and we want to test version mismatches, we can change/increase minSdk version of our testing/development apps (`analyticsexample`, `bitmovinanalyticsexample`) to latest SDK version. I have not done this as I wanted to stay with minSdk version we had before, and that was `v16` in all modules.